tools/travis/script.sh

```diff
@@ -102,6 +102,38 @@ run_sql_test()
   case "${CI_MYSQL_VERSION}" in
     mysql-8.0)
       n_processors=1
+      # Remove not supported functionality test
```

Comment:
Could you also add TODO comment to remove these `rm`s as soon as possible?

Reply:
Thanks, I've added comment.